example/dirtylarry.gui_script

```diff
@@ -5,9 +5,9 @@ function init(self)
 
 	dirtylarry.checkbox("checkbox").set_checked(true)
 	dirtylarry.radio("radio2").set_selected(true)
-	dirtylarry.input("input", gui.KEYBOARD_TYPE_DEFAULT, nil, nil, { empty_text = "EMPTY, MAX 8 CHARS" })
+	dirtylarry.input("input", gui.KEYBOARD_TYPE_DEFAULT, nil, nil, { empty_text = "EMPTY, MAX 8 CHARS" }).set_use_mark_text(false)
```

Comment:
I think we should use the config table instead of adding a new function. Like this:

    { empty_text = "EMPTY, MAX 8 CHARS", use_marked_text = false }